### PR TITLE
Added book lot card title and renamed book demand title 

### PR DESCRIPTION
**Pull request by NaimulHasanFahim**

### CODE REVIEW

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js [hunk 1]
```diff
@@ -325,20 +325,20 @@ const TITLE_LOCALES_BN = {
...
-    'en': 'Submitted Book Demand',
-    'bn': 'বইয়ের চাহিদা জমা হয়েছে'
+    'en': 'Submitted School Book Demand',
+    'bn': 'বিদ্যালয়ের বইয়ের চাহিদা জমা হয়েছে'
   },
   'admin.dashboard.book.distribution.progress.approved' : {
-    'en': 'Approved Book Demand',
-    'bn': 'অনুমোদিত বইয়ের চাহিদা'
+    'en': 'Approved School Book Demand',
+    'bn': 'বিদ্যালয়ের অনুমোদিত বইয়ের চাহিদা'
```
Comment (osman-goni-cse) [inline frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js:333]:
```suggestion
    'bn': 'বিদ্যালয়ের বইয়ের চাহিদা অনুমোদন হয়েছে'
```

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js [hunk 2]
```diff
@@ -325,20 +325,20 @@ const TITLE_LOCALES_BN = {
...
   },
   'admin.dashboard.book.distribution.progress.pending' : {
-    'en': 'Pending Book Demand',
-    'bn': 'পেন্ডিং বইয়ের চাহিদা'
+    'en': 'Pending School Book Demand',
+    'bn': 'বিদ্যালয়ের পেন্ডিং বইয়ের চাহিদা'
```
Comment (osman-goni-cse) [inline frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js:337]:
```suggestion
    'bn': 'বিদ্যালয়ের বইয়ের চাহিদা পেন্ডিং আছে'
```

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js [hunk 3]
```diff
@@ -325,20 +325,20 @@ const TITLE_LOCALES_BN = {
...
   },
   'admin.dashboard.book.distribution.progress.rejected' : {
-    'en': 'Rejected Book Demand',
-    'bn': 'প্রত্যাখ্যাত বইয়ের চাহিদা'
+    'en': 'Rejected School Book Demand',
+    'bn': 'বিদ্যালয়ের প্রত্যাখ্যাত বইয়ের চাহিদা'
```
Comment (osman-goni-cse) [inline frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js:341]:
```suggestion
    'bn': 'বিদ্যালয়ের বইয়ের চাহিদা প্রত্যাখ্যান হয়েছে'
```

**Review by osman-goni-cse — CHANGES_REQUESTED** [review]

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js (hunk 1 above)
Comment (NaimulHasanFahim) [reply frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js]: done

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js (hunk 2 above)
Comment (NaimulHasanFahim) [reply frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js]: done

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js (hunk 3 above)
Comment (NaimulHasanFahim) [reply frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js]: done

**Review by osman-goni-cse — APPROVED** [review]
LGTM

**Pull request merged**